src/utils/ws_utils.py

```diff
@@ -1,3 +1,9 @@
+from src.utils.config import config
```

Comment:
I think this should be a class and it should basically mimic the standard workspace client calls.